flink-runtime/src/main/scala/org/apache/flink/runtime/minicluster/FlinkMiniCluster.scala

```diff
@@ -366,6 +428,28 @@ abstract class FlinkMiniCluster(
   //                          Utility Methods
   // --------------------------------------------------------------------------
 
+//  /**
+//    * Waits for the leading ResourceManager to be registered at the JobManager
+//    */
+//  def waitForResourceManagerToBeRegistered(): Unit = {
+//    waitForResourceManagerToBeRegistered(timeout)
+//  }
+//
+//
+//  def waitForResourceManagerToBeRegistered(timeout: FiniteDuration): Unit = {
+//
+//    future {
+//      resourceManagerActors foreach {
+//        list => list.foreach(rm => rm ! NotifyWhenResourceManagerRegistered)
+//      }
+//
+//      rece
+//    }
+//
+//    Await.ready()
+//
+//  }
```

Comment:
Good catch. Commented this utility function because it was not necessary anymore.